main.js

```diff
@@ -826,6 +826,26 @@ function extendNode(nodeID, nodeInfo, callback) {
     if (!count && callback) callback();
 }
 
+/*
+* Verify that we don't have duplicate valueIds (same node, instance and index but different label)
+* Can happen when the ozwave library changes labels for devices between releases
+*/
+function cleanupValueId(nodeID, comClass, valueId) {
+    const channelID = calcName(nodeID, comClass);
+    
+    for (var i in objects) {
+        if (!objects.hasOwnProperty(i)) continue;
+        if (i.startsWith(channelID + '.')) {
+            if (objects[i].native && objects[i].native.instance === valueId.instance && objects[i].native.index === valueId.index &&
+            objects[i].common.name !== valueId.label) {
+                adapter.log.info('remove obsolete state:' + objects[i]._id);
+                delObjects([objects[i]]);
+                break;
```

Comment:
Since this is now its own function, `return` seems to make more sense, right?

Reply:
True, adjusted